pyproject.toml

```diff
@@ -14,7 +14,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
-hikari = ">=2.0.0.dev108"
+hikari = { git = "https://github.com/hikari-py/hikari" }
```

Comment:
Should fix this

Reply:
done. Opened a draft PR so i could change it lol.